interface/src/PerformanceManager.cpp

```diff
@@ -116,10 +127,12 @@ void PerformanceManager::applyPerformancePreset(PerformanceManager::PerformanceP
 
             DependencyManager::get<LODManager>()->setWorldDetailQuality(WORLD_DETAIL_LOW);
 
-        break;
-        case PerformancePreset::UNKNOWN:
+            break;
+        case
```

Comment:
Why the newline here?

Reply:
Good question. I must have been tired or something. I'll push a fix now.